src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -434,14 +851,65 @@ .maxstack  8
+                //         c.Field(__arglist(1, 2));
+                Diagnostic(ErrorCode.ERR_UnsupportedCallingConvention, "Field").WithArguments("delegate*<void>").WithLocation(6, 11),
+                // (7,11): error CS8795: The calling convention of 'delegate*<void>' is not supported the language.
+                //         c.Field(1, 2, 3);
+                Diagnostic(ErrorCode.ERR_UnsupportedCallingConvention, "Field").WithArguments("delegate*<void>").WithLocation(7, 11),
+                // (8,11): error CS8795: The calling convention of 'delegate*<void>' is not supported the language.
+                //         c.Field();
+                Diagnostic(ErrorCode.ERR_UnsupportedCallingConvention, "Field").WithArguments("delegate*<void>").WithLocation(8, 11),
+                // (9,11): error CS8795: The calling convention of 'delegate*<void>' is not supported the language.
+                //         c.Field = c.Field;
+                Diagnostic(ErrorCode.ERR_UnsupportedCallingConvention, "Field").WithArguments("delegate*<void>").WithLocation(9, 11),
+                // (9,21): error CS8795: The calling convention of 'delegate*<void>' is not supported the language.
+                //         c.Field = c.Field;
+                Diagnostic(ErrorCode.ERR_UnsupportedCallingConvention, "Field").WithArguments("delegate*<void>").WithLocation(9, 21),
+                // (10,11): error CS8795: The calling convention of 'delegate*<void>' is not supported the language.
+                //         c.Prop();
+                Diagnostic(ErrorCode.ERR_UnsupportedCallingConvention, "Prop").WithArguments("delegate*<void>").WithLocation(10, 11),
+                // (11,11): error CS8795: The calling convention of 'delegate*<void>' is not supported the language.
+                //         c.Prop = c.Prop;
+                Diagnostic(ErrorCode.ERR_UnsupportedCallingConvention, "Prop").WithArguments("delegate*<void>").WithLocation(11, 11),
+                // (11,20): error CS8795: The calling convention of 'delegate*<void>' is not supported the language.
+                //         c.Prop = c.Prop;
+                Diagnostic(ErrorCode.ERR_UnsupportedCallingConvention, "Prop").WithArguments("delegate*<void>").WithLocation(11, 20)
+            );
 
             var c = comp.GetTypeByMetadataName("C");
             var prop = c.GetProperty("Prop");
 
             VerifyFunctionPointerSymbol(prop.Type, CallingConvention.FastCall,
                 (RefKind.None, IsVoidType()));
+
+            Assert.True(prop.Type.HasUseSiteError);
+
+            var field = c.GetField("Field");
+
+            var type = (FunctionPointerTypeSymbol)field.Type;
+            VerifyFunctionPointerSymbol(type, CallingConvention.ExtraArguments,
+                (RefKind.None, IsVoidType()));
+
+            Assert.True(type.HasUseSiteError);
+            Assert.True(type.Signature.IsVararg);
```

Comment:
>Assert.True(type.Signature.IsVararg); [](start = 12, length = 37)

Is there any negative tests for this property. I.e. we expect false and assert it is false.

Reply:
Will add to `CommonVerifyFunctionPointer`

---
In reply to: [413424009](https://github.com/dotnet/roslyn/pull/43355#discussion_r413424009) [](ancestors = 413424009)